vdev/Cargo.toml

```diff
@@ -35,5 +35,5 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.95"
 serde_yaml = "0.9.19"
 sha2 = "0.10.6"
-tempfile = "3.4.0"
+tempfile = "3.5.0"
```

Comment:
I'm curious why this needed updating? It looks like it didn't change the actual crate version, so it's just reflecting what we're already using.

Reply:
I could be wrong but when I tried a merge master it used tempfile 3.5.0. So I tried matching the version. I assume I need to have these in the `vdev/Cargo.toml` match the one in the repo level.